cms/standard_pages/tests/test_models.py

```diff
@@ -0,0 +1,115 @@
+from django.test import TestCase, override_settings
+from wagtail.test.utils import WagtailTestUtils
+
+from cms.standard_pages.tests.factories import IndexPageFactory, InformationPageFactory
+
+
+class IndexPageTestCase(WagtailTestUtils, TestCase):
+    """Test IndexPage model properties and methods."""
+
+    def setUp(self):
+        self.page = IndexPageFactory(
```

Comment:
suggestion: revisiting this, can you rename this to `self.index_page` for full clarity?

Reply:
[Test improvements 2.0](https://github.com/ONSdigital/dis-wagtail/pull/78/commits/612a1db5994dced67b02a37db7f3c3518885204b)